content/en/blog/_posts/2023-12-20-disabling-in-tree-cloud-provider-goes-beta.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: blog
+title: "Removing In-Tree Cloud Providers Graduates to Beta Status"
+date: 2023-12-20
+slug: removing-in-tree-cloud-providers-beta-status
+---
+
+**Authors:** Michael McCune, Andrew Sy Kim
+
```

Comment:
Make sure to state the main point.
For example:
```suggestion
For Kubernetes v1.29, you need to use additional components to integrate your Kubernetes cluster
with a cloud infrastructure provider. By default, Kubernetes v1.29 does not run with the legacy,
compiled-in cloud provider integrations enabled.

```

Reply:
ii've added this to the beginning.